app/Http/Controllers/API/V1/Auth/RegistrationContoller.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers\API\V1\Auth;
+
+use AElnemr\RestFullResponse\CoreJsonResponse;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\RegistrationRequest;
+use App\Http\Resources\UserResource;
+use App\Models\User;
+use App\Models\UserProfile;
+use Illuminate\Http\Request;
+
+class RegistrationContoller extends Controller
+{
+    use CoreJsonResponse;
+
+    public function register(RegistrationRequest $request) {
+        $user = User::create($request->only(['name', 'email', 'password']));
+        $user->profile()->create($request->only(['bio']));
+
+        return $this->created((new UserResource($user))->resolve());
```

Comment:
Why you don't return the token in registration?!
And what if you have any problem in UserProfile model,
You will save the user data in users table without profile, 
Try to check Transaction